src/scripts/main.js

```diff
@@ -1,3 +1,7 @@
 'use strict';
 
-// write your code here
+const tableRows = document.querySelectorAll('tr');
+
+tableRows.forEach(row => {
+  row.children[row.children.length - 1].before(row.children[1].cloneNode(true));
```

Comment:
good solution, but in general looks beter if we not calculate all time, and in future if someone will want to copy some other column it will be easier if he will something like 'copyColumn', 'pastePosition'

just sugestion